python/src/OTtestcode_doc.i.in

```diff
@@ -0,0 +1,188 @@
+%feature("docstring") OT::Test::assert_almost_equal
+"Raises an exception if two objects are not equal up to desired tolerance.
+
+Similar to numpy.testing.assert_allclose.
+
+Parameters
+----------
+a, b : float, Point, Matrix, Distribution, Mesh, Field or ProcessSample
+    Elements to compare
+rtol : float
+    Relative tolerance, default is 1e-5
+atol : float
+    Absolute tolerance, default is 1e-8
+errMsg : str
+    Error message emitted when comparison fails
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.testing as ott
+>>> dist = ot.Uniform(0.0, 1.0)
+>>> ott.assert_almost_equal(dist.getMean(), [0.5])
+"
+
+// ---------------------------------------------------------------------
+    
+%feature("docstring") OT::Test::DistributionChecker
+"Automates distribution tests.
+
+It helps performing tests to validate :class:`~openturns.Distribution` services:
+
+- Comparison operators
+- PDF: analytical PDF vs finite-difference estimate from CDF
```

Comment:
```suggestion
- CDF: analytical PDF vs finite-difference estimate from CDF
```

Reply:
depends, most time we consider the CDF being the reference